phpBB/phpbb/extension/extension_interface.php

```diff
@@ -27,6 +27,13 @@
 	public function is_enableable();
 
 	/**
+	* Indicate why the extension can not be enabled.
+	*
+	* @return string|array with actual processed language string(s) (not language keys), or empty
+	*/
+	public function not_enableable_reason();
```

Comment:
Open to suggestions. Maybe just `not_enableable`.

Don't agree with the return format: what would be the meaning of [] or ''?
All truish values should mean the same, and all falsish values as well.